challenges/objects-arrays.js

```diff
@@ -8,48 +8,129 @@
-{"id":3,"first_name":"Lambert","university":"Marian College","email":"lparham2@techcrunch.com"},
-{"id":4,"first_name":"Modestine","university":"International Medical & Technological University","email":"mdolder3@symantec.com"},
-{"id":5,"first_name":"Chick","university":"Sultan Salahuddin Abdul Aziz Shah Polytechnic","email":"camorts4@google.com.au"},
-{"id":6,"first_name":"Jakob","university":"Fachhochschule Rosenheim, Hochschule für Technik und Wirtschaft","email":"jharken5@spiegel.de"},
-{"id":7,"first_name":"Robbi","university":"Salem University","email":"rbrister6@redcross.org"},
-{"id":8,"first_name":"Colline","university":"Coastal Carolina University","email":"cbrosh7@alibaba.com"},
-{"id":9,"first_name":"Michail","university":"Universidad Católica de Ávila","email":"mrome8@shinystat.com"},
-{"id":10,"first_name":"Hube","university":"Universitat Rovira I Virgili Tarragona","email":"hlethbrig9@foxnews.com"}]
+const graduates = [{
+        "id": 1,
+        "first_name": "Cynde",
+        "university": "Missouri Southern State College",
+        "email": "ctorry0@macromedia.com"
+    },
+    {
+        "id": 2,
+        "first_name": "Saundra",
+        "university": "The School of the Art Institute of Chicago",
+        "email": "swhal1@state.gov"
+    },
+    {
+        "id": 3,
+        "first_name": "Lambert",
+        "university": "Marian College",
+        "email": "lparham2@techcrunch.com"
+    },
+    {
+        "id": 4,
+        "first_name": "Modestine",
+        "university": "International Medical & Technological University",
+        "email": "mdolder3@symantec.com"
+    },
+    {
+        "id": 5,
+        "first_name": "Chick",
+        "university": "Sultan Salahuddin Abdul Aziz Shah Polytechnic",
+        "email": "camorts4@google.com.au"
+    },
+    {
+        "id": 6,
+        "first_name": "Jakob",
+        "university": "Fachhochschule Rosenheim, Hochschule für Technik und Wirtschaft",
+        "email": "jharken5@spiegel.de"
+    },
+    {
+        "id": 7,
+        "first_name": "Robbi",
+        "university": "Salem University",
+        "email": "rbrister6@redcross.org"
+    },
+    {
+        "id": 8,
+        "first_name": "Colline",
+        "university": "Coastal Carolina University",
+        "email": "cbrosh7@alibaba.com"
+    },
+    {
+        "id": 9,
+        "first_name": "Michail",
+        "university": "Universidad Católica de Ávila",
+        "email": "mrome8@shinystat.com"
+    },
+    {
+        "id": 10,
+        "first_name": "Hube",
+        "university": "Universitat Rovira I Virgili Tarragona",
+        "email": "hlethbrig9@foxnews.com"
+    }]
 
 /* Request 1: Create a new array called universities that contains all the universities in the graduates array.  
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
 const universities = [];
+const universitiesList = graduates.map(university => university.university)
```

Comment:
Nice approach to do the `.map()` directly into a variable and then using that variable to `sort` and `push`. Good and minimal approach 👍